src/__tests__/index.js

```diff
@@ -192,3 +204,46 @@ test('forwards props when the GlamorousComponent.rootEl is known', () => {
   )
   expect(ui).toMatchSnapshotWithGlamor()
 })
+
+test('renders a component with theme properties', () => {
+  const Comp = glamorous.div(
+    {
+      color: 'red',
+    },
+    (props, theme) => ({padding: theme.padding}),
+  )
+  expect(
+    render(<Comp theme={{padding: '10px'}} />),
+  ).toMatchSnapshotWithGlamor()
+})
+
+test('passes an updated theme when theme prop changes', () => {
+  const Comp = glamorous.div(
+    {
+      color: 'red',
+    },
+    (props, theme) => ({padding: theme.padding}),
+  )
+  const wrapper = mount(<Comp theme={{padding: '10px'}} />)
+  expect(toJson(wrapper)).toMatchSnapshotWithGlamor(`enzyme.mount`)
+  wrapper.setProps({theme: {padding: '20px'}})
+  expect(toJson(wrapper)).toMatchSnapshotWithGlamor(`enzyme.mount`)
```

Comment:
As noted above, because we use [the serializer](https://github.com/paypal/glamorous/blob/14970c1bb59d1408d3066d209dafa356b5ce3173/package.json#L97-L99) this could simply be:

```javascript
expect(wrapper).toMatchSnapshotWithGlamor('enzyme.mount')
```